### ISSUE TITLE
I18n does not apply.

### ISSUE THREAD

**Issue by rnrnstar2** [aws-amplify/amplify-ui#1212]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

React

### Which UI component?

Authenticator

### How is your app built?

next.js

### Please describe your bug.

Your code is on the way. To log in, enter the code we emailed to p***@g***. It may take a minute to arrive.

The email will be masked and displayed when you verify the verification code.
Then, it cannot be converted to Japanese.
Please correct this.

### What's the expected behaviour?

Do not mask and display email.

### Help us reproduce the bug!

![Uploading スクリーンショット 2022-01-29 15.47.05.png…]()

### Code Snippet

```typescript
// Put your code below this line.

```

**Comment by rnrnstar2** [comment]
I also created a field to enter the survey in signup, but it doesn't work.
const [questionForm, setQuestionForm] = useState <string> ('');
Running setQuestionForm will erase existing data such as username and email.
Can't you use useState?

https://user-images.githubusercontent.com/39724729/151654785-c8301fb4-df7d-4bd0-9bec-b84f49047802.mov

**Comment by rnrnstar2** [comment]
Another thing, when I log out and the login screen is displayed, the placeholder of username and password is not displayed.

<img width="635" alt="スクリーンショット 2022-01-29 18 03 40" src="https://user-images.githubusercontent.com/39724729/151654971-4f93251a-60dd-4cdb-b544-cf0edae00b10.png">

**Comment by ErikCH** [comment]
Hi @rnrnstar2 !

I have pushed up a fix for the translations in #1357 for the confirmSignUp page so you can now translate them. It should be in our next release.

As for your other issues. Are they still occurring? I was not able to reproduce the issue on sign up placeholder text disappearing after signing out.

Here is an example overriding sign up fields. And adding your own.  

https://ui.docs.amplify.aws/components/authenticator#sign-up-fields

Creating your own fields should not erase other fields. You should use `formData` from the `validateCustomSignUp` to add in your own additional validation.

```
 services={{
        async validateCustomSignUp(formData) {
          if (!formData.acknowledgement) {
            return {
              acknowledgement: 'You must agree to the Terms & Conditions',
            };
          }
        },
      }}
```

I'm going to close this for now. But please reopen a different issue for these other problems if they exist again.

**Comment by rnrnstar2** [comment]
Thank you for you help :)
I will try them when it released.
